index.html

```diff
@@ -33,9 +33,149 @@
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+    <div class="grid-container-another">
+        <div class="about-me">
+            <h2 class="about">About Me</h2>
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
+        </div>
+        <div class="grid-item-another">
+            <img class="skills languagelogo" src="./icon-languages@2x.png" alt="language logo">
+            <h3 class="language">Languages</h3>
+            <div class="mini-flex-container">
+                <label class="cli mini-item">Javascript</label>
+                 <label class=" mini-item">HTML</label>
+                 <label class=" mini-item">Ruby</label> 
+                 <label class=" mini-item">CSS</label>  
+             </div>
+        </div>
+        <div class="grid-item-another">
+            <img class="skills framelogo" src="./icon-frameworks@2x.png" alt="framework logo">
+            <h3 class="language">Frame Work</h3>
+            <div class="mini-flex-container">
+                <label class="cli mini-item">React.js</label>
+                 <label class=" mini-item">Rspec</label>
+                 <label class=" mini-item">capybara</label>  
+             </div>
+        </div>
+        <div class="grid-item-another">
+            <img class="skills skillslogo" src="./icon-skills@2x.png" alt="skills logo">
+            <h3 class="language">Skills</h3>
+            <div class="mini-flex-container">
+               <label class="cli mini-item">CLI</label>
+                <label class="web mini-item">web developement</label>
+                <label class="api mini-item">API design</label>  
+            </div>
+        </div>
+    </div>
     
+        <h4 class="interested">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+        <div class="container">
+            <form action="/action_page.php">
+              
+              <input type="text" class="email" name="firstname" placeholder="Your name">
+          
+              
+              <input type="text" class="email" name="email" placeholder="Email Address">
+          
+              
+              <textarea id="subject" name="subject" placeholder="Hey i'd like to connect..." ></textarea>
+        
+            </form>
+            <button class="getintouch">Get in touch</button>
+          </div>
+</section>
+  <footer class="footer">
+    <h4 class="emails">affushaik9493@gmail.com</h4>
+    <ul class="footer-sec socialmediaicons">
+        <li><a href="https://www.twitter.com/tajulafreen"><i class="fa fa-twitter"></i></a></li>
```

Comment:
- [ ] This section features are for the next milestones and would not be part of the reviewed files. :+1: